person/forms.py

```diff
@@ -0,0 +1,175 @@
+import logging
+
+from django import forms
+from django.forms import ModelForm, CharField
+
+from core.helper import form_utils
+from person.models import CofkUnionPerson
+
+log = logging.getLogger(__name__)
+
+calendar_date_choices = [
+    ("", 'Unknown'),
+    ("G", 'Gregorian'),
+    ("JM", 'Julian (year starting 25th Mar)'),
+    ("JJ", 'Julian (year starting 1st Jan)'),
+    ("O", 'Other'),
+]
+
+
+# def calendar_date
+
+
+class PersonForm(ModelForm):
+    gender = CharField(required=False,
+                       widget=forms.RadioSelect(choices=[
+                           ('M', 'Male'),
+                           ('F', 'Female'),
+                           ('', 'Unknown or not applicable'),
+                       ])
+                       )
+
+    is_organisation = form_utils.ZeroOneCheckboxField(required=False, initial='1', )
+    roles_titles = forms.CharField(required=False,
+                                   widget=forms.CheckboxSelectMultiple(
+                                       choices=[
+                                           ('5', 'Antiquary'),
```

Comment:
is it a good idea to hard-code these?
is it not in a table?

Reply:
For now should be fine, 
If other code need to use those choices, I will extract as constant variable.
If there have requirement that User can change those choice in website, I will extract to those values to DB